modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/IgniteSqlFunctions.java

```diff
@@ -45,6 +50,15 @@
  * Ignite SQL functions.
  */
 public class IgniteSqlFunctions {
+    private static final int DATE_MIN_INTERNAL = (int) TypeUtils.toInternal(SchemaUtils.DATE_MIN, LocalDate.class);
```

Comment:
TypeUtils.toInternal - already deprecated method, you need to avoid it`s usage

Reply:
Fixed, thanks.